show/muxcable.py

```diff
@@ -336,3 +339,43 @@ def config(port, json_output):
             click.echo(tabulate(print_data, headers=headers))
 
         sys.exit(CONFIG_SUCCESSFUL)
+
+
+@muxcable.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+def berinfo(port, target):
+
+    if os.geteuid() != 0:
+        click.echo("Root privileges are required for this operation")
+        sys.exit(EXIT_FAIL)
+    import sonic_y_cable.y_cable
+    res = sonic_y_cable.y_cable.get_ber_info(port, target)
+    if res == False or res == -1:
+        click.echo("Unable to fetch ber info")
+        sys.exit(EXIT_FAIL)
+    headers = ['Lane1', 'Lane2']
+    lane_data = []
+    lane_data.append(res)
+    click.echo(tabulate(lane_data, headers=headers))
+    sys.exit(EXIT_SUCCESS)
+
+
+@muxcable.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+def eyeinfo(port, target):
+
```

Comment:
Please add a docstring to be used as a brief help message

Reply:
fixed